lib/components/src/Accordion/stories/AccordionBody.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { AccordionBody } from '../AccordionBody';
+
+// eslint-disable-next-line import/order
+import type { Story, Meta } from '@storybook/react';
+import type { AccordionBodyProps } from '../AccordionBody';
+
+export default {
+  title: 'Basics/Accordion/AccordionBody',
+  component: AccordionBody,
+  parameters: {
+    layout: 'padded',
+  },
+} as Meta;
+
+const Template: Story<AccordionBodyProps> = (args) => (
```

Comment:
There's a new type ComponentMeta which is quite convenient, you don't need to export the props anymore:

```suggestion
import type { ComponentStory, Meta } from '@storybook/react';

import { AccordionBody } from '../AccordionBody';

export default {
  title: 'Basics/Accordion/AccordionBody',
  component: AccordionBody,
} as Meta;

const Template: ComponentStory<typeof AccordionBody> = (args) => (
```